hid-uclogic.c

```diff
@@ -948,6 +1019,23 @@ static int uclogic_enable_tablet(struct hid_device *hdev, __le16 **pbuf)
 	return rc;
 }
 
+static void uclogic_fill_placeholders(__u8* prdesc, unsigned int rsize, s32* params)
+{
+	__u8 *p;
+	s32 v;
+	for (p = prdesc;
+	     p <= prdesc + rsize -4;) {
+		if (p[0] == 0xFE && p[1] == 0xED && p[2] == 0x1D &&
+		    p[3] < UCLOGIC_PH_ID_NUM) {
```

Comment:
If we are passing `params` as the argument, we better pass number of its elements as well, instead of using a global macro.